from bert import exceptions

class BertConfigError(exceptions.BertException):
    pass

